bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/auxfile/AuxFileEditor.scala

```diff
@@ -27,7 +27,7 @@ object AuxFileEditor {
       lab.text = size.toString
   })
 
-  private val Format = new SimpleDateFormat("MM/dd/yy HH:mm:ss") {
+  private val Format = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss") {
```

Comment:
It would be nice to increase the width but Column's widths is not that trivial in Swing, There is a utility trait https://github.com/gemini-hlsw/ocs/blob/develop/bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/TableColumnsAdjuster.scala but i don't know how applicable is for your particular case